tests/admin_widgets/false_timezone_warning_templates/admin/base.html

```diff
@@ -0,0 +1,88 @@
+{% load i18n admin_static %}<!DOCTYPE html>
```

Comment:
I could minimize it as much as this:
```
<html>
<head>
{% block extrahead %}{% endblock %}
</head>
<body>
{% block content %}
{{ content }}
{% endblock %}
</body>
</html>
```

But I fear that the test may quickly return false positives because of other issues causing the JavaScript to not run - in that case, the test would return as OK because there are no `.timezonewarning` elements. An example would be if the Django admin at some point doesn't put the JavaScript in `extrahead`, but in the page footer. This can happen with the full file too, of course, but hopefully not as quickly.

One could add a second test case that tests that the warnings are indeed shown with mostly the same template, except that it includes the data attribute. Something like `test_false_timezone_warnings_safety_check`... not sure if this makes sense though.

Would this comment on the top of the file be correct?
`{# Intentionally missing data-admin-utc-offset in body tag to test for false timezone warnings. #}`